features/support/e2e.rb

```diff
@@ -10,3 +10,7 @@
 After("@e2e") do
   Capybara.server_port = nil
 end
+
+Before do
+  Rails.application.load_seed
+end
```

Comment:
The cucumber tests are set up to work with seed data. However we don't want to seed the db, since it's also used for rspec tests. So instead this sets the data up on a test-by-test basis.